extensions/cornerstone/src/commandsModule.ts

```diff
@@ -11,29 +11,53 @@ import {
   ReferenceLinesTool,
 } from '@cornerstonejs/tools';
 import { ServicesManager } from '@ohif/core';
+import { ContextMenu } from '@ohif/ui';
 
 import { getEnabledElement as OHIFgetEnabledElement } from './state';
 import CornerstoneViewportDownloadForm from './utils/CornerstoneViewportDownloadForm';
+import { connectToolsToMeasurementService } from './initMeasurementService';
 import callInputDialog from './utils/callInputDialog';
 import { setColormap } from './utils/colormap/transferFunctionHelpers';
 import toggleMPRHangingProtocol from './utils/mpr/toggleMPRHangingProtocol';
 import toggleStackImageSync from './utils/stackSync/toggleStackImageSync';
+import defaultContextMenu from './defaultContextMenu';
 
-const commandsModule = ({ servicesManager }) => {
+const commandsModule = ({ servicesManager, commandsManager }) => {
   const {
-    ViewportGridService,
-    ToolGroupService,
+    viewportGridService,
+    toolGroupService,
     cineService,
     toolbarService,
-    UIDialogService,
-    CornerstoneViewportService,
-    HangingProtocolService,
-    UINotificationService,
+    uiDialogService,
+    cornerstoneViewportService,
+    hangingProtocolService,
+    uiNotificationService,
+    customizationService,
+    displaySetService,
+    measurementService,
```

Comment:
same for these, let's have separate PRs for group of services, the approach to touch anything we see first makes the scope of this PR much larger for review (either for reviewer or for the reader later), and also doesn't achieve the end goal of changing the service name fully

Reply:
In addition to above, it makes testing a lot harder! if we follow what you are doing right now, then we should test rigorously all components for each PR (segmentation, measurements, hangign protocol, etc. etc.). Please don't do this, and let's have separate PRs for each service

Reply:
The service isn't changing, it is just using typed access to the service.  When touching a file, I think we should use the new naming/typing because it makes it much easier to get the design right.